data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/reconciliation/response/LookupKeyByIdResponse.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "record",
+  "name": "LookupKeyByIdResponse",
+  "namespace": "net.corda.data.crypto.wire.ops.reconciliation.response",
```

Comment:
I think this looks fine. 

Only thing, not really sure if we should be binding it with reconciliation (i.e. namespace). It could be consumed by other things in the future.

Perhaps some `sync` in the namespace would make sense since the form of it implies sync since it carries the error as well? (I think when it comes to types over Kafka, i.e. async we handle errors under the hood and then the user doesn't need to handle it themselves?)

Reply:
I changed `reconciliation` to `sync`. In the future we can add other operations.